code/modules/power/tesla/energy_ball.dm

```diff
@@ -0,0 +1,94 @@
+/obj/singularity/energy_ball
+	name = "Energy Ball"
+	icon = 'icons/obj/tesla_engine/energy_ball.dmi'
+	icon_state = "energy_ball"
+	pixel_x = -32
+	pixel_y = -32
+	current_size = STAGE_TWO
+	move_self = 1
+	grav_pull = 0
+	contained = 0
+	density = 1
+
+/obj/singularity/energy_ball/process()
+	pixel_x = 0
+	pixel_y = 0 // Lining up the beams properly.
+	tesla_zap(src, 3, 3476520)
```

Comment:
this is a magic number, it's a number used with no explanation as to why, thus it appears "magic", truer magic numbers or constants actually appear to do magic (like that inverse square root one) but this still counts for what we disallow as "magic"